### PR TITLE
Update and rename Policy_for_OBO_namespace_and_associated_PURL_reques…

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
…ts.md to Submit_a_new_ontology.md

addresses #1246

### CODE REVIEW

**Review by nlharris — COMMENTED** [review]
My suggestion, as per my comment about how-do-i-register-my-ontology.md, is to keep this page's title and focus on the idspace and purl stuff, and meanwhile change how-do-i-register-my-ontology.md to describe how to register a new ontology. What do you think?

**Comment by nicolevasilevsky** [comment]
That sounds good, Nomi! 
Do you think we should leave the title on this page as it was?

I updated the FAQ page on 'how to register my ontology', with the simple steps. This page currently includes duplicate info, but it is an expanded version with more details. Do you think the info should be on two separate pages?

Also - I think that just having 'submit an ontology' under FAQs isn't very intuitive. I think it should be under FAQ, but maybe somewhere else more prominent too.

**Comment by nicolevasilevsky** [comment]
Oh, I see there is info under "Review Process Guidelines" too, with simple steps:
http://obofoundry.org/docs/ReviewProcessGuidelines.html

This info seems to be duplicated in a lot of places.

Not sure the best way to fix it!

**Comment by nlharris** [comment]
Yes, I agree! I said in #1283, "I think maybe it would make more sense to make this page (faq/how-do-i-register-my-ontology.md) the simple instructions, and make it easier to find (not just in the FAQ menu)." We can put a prominent link to it on the revised home page (see #241) and also add it to the "Participate" menu (as suggested in #1246).

File: docs/Submit_a_new_ontology.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
-id: Policy_for_OBO_namespace_and_associated_PURL_requests
-title: Policy for OBO IDSPACE and associated PURL requests
+id: Submit_a_new_ontology
```
Comment (nlharris) [inline docs/Submit_a_new_ontology.md:3]: I think we should change this (and the title) to be something like "OBO Foundry membership requirements and technical details".

File: docs/Submit_a_new_ontology.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+title: Submit a New Ontology to the OBO Foundry
 ---
 
 Please raise any questions or concerns on our GitHub tracker: https://github.com/OBOFoundry/OBOFoundry.github.io/issues
```
Comment (nlharris) [inline docs/Submit_a_new_ontology.md:7]: Change to "For a simple overview of how to submit your ontology for consideration, please go [here](URL of new submit an ontology page)"

Comment (nlharris) [reply docs/Submit_a_new_ontology.md:7]: I think we should move the Background section to the end.

**Review by nlharris — COMMENTED** [review]
Let's resolve some of these suggestions and then we can iterate to refine this page more.

File: docs/Submit_a_new_ontology.md (hunk 1 above)
Comment (nicolevasilevsky) [reply docs/Submit_a_new_ontology.md]: i made this change

File: docs/Submit_a_new_ontology.md (hunk 2 above)
Comment (nicolevasilevsky) [reply docs/Submit_a_new_ontology.md]: i made these changes.

the URL will need to be updated though after the new submit an ontology page PR is merged #1283

**Review by nlharris — APPROVED** [review]

**Comment by jamesaoverton** [comment]
Renaming this file required changing a bunch of links across the website. It would have been cleaner and more clear to make all those changes in a single Pull Request instead of several.

Failing that, those others PRs #1302 #1304 #1305 should have mentioned this PR, making it clear that the changes are linked.

**Comment by jamesaoverton** [comment]
More generally, renaming this file broke any links that anyone had to these instructions anywhere else on the Web. I have to search through the various documentation that I've written over the years and see if I need to change anything. We don't know who else might have linked to this page.

The original issue didn't discuss renaming the file. This PR was created and merged before I had a chance to raise these concerns.

We have a strict policy of maintaining PURLs for OBO terms. I think we should also be serious about keeping the links to our documentation stable.

@cmungall Is it possible to tell GitHub pages to redirect to the new path?

**Comment by matentzn** [comment]
I would re-instate the old files with a link to the new place for now..

**Comment by nicolevasilevsky** [comment]
I started working on this on PR #1309 (work in progress)

**Comment by cmungall** [comment]
I agree with @jamesaoverton 

IF we are to rename, I think then we should give this a shorter more memorable URL, no need to put an essay in the URL :-). Just "/register" would be better.

I don't know exactly how to do a redirect; may require a plugin which I don't think we have control over if we use GH for hosting
https://github.com/jekyll/jekyll-redirect-from

We've done this kind of thing in the past with javascript which is icky

Of course you can always do a human-level redirect, e.g. static text that says "page moved to X".. but we should follow WWW standards and use HTTP codes

**Pull request merged**